wcfsetup/install/files/lib/acp/form/UserAddForm.class.php

```diff
@@ -124,6 +128,35 @@ class UserAddForm extends UserOptionListForm
      * @var array
      */
     public $optionTree = [];
+    public ?AttachmentHandler $attachmentHandler;
```

Comment:
This makes no sense. The handler is either set or not, at no point is a `value` being assigned. Please keep in mind that typed properties without an initializer are not set at all, which is different to untyped properties that will implicitly initialized with `null`.